front/src/modules/ui/view-bar/components/SortOrFilterChip.tsx

```diff
@@ -67,7 +67,7 @@ function SortOrFilterChip({
     <StyledChip isSort={isSort}>
       {Icon && (
         <StyledIcon>
-          <Icon />
+          <Icon size={14} />
```

Comment:
Can you use the theme for that ? I think it's `size.sm` ?

Reply:
See the `useTheme` hook

Reply:
sure

Reply:
sure, fixed